package.json

```diff
@@ -28,16 +28,16 @@
     "lint:root:eslint": "eslint \"**/*.{js,cjs,ts,tsx}\" --max-warnings=0",
     "prettier": "prettier . --cache --check",
     "test": "jest",
-    "cypress": "cypress open",
-    "cypress:run": "cypress run",
+    "cypress": "cypress open --e2e --browser firefox",
```

Comment:
Starting FF automatically when launching Cypress locally. Doesn't prevent opening the main Cypress window unfortunately.